doc/changes/changes_4.0.1.md

```diff
@@ -0,0 +1,20 @@
+# Exasol Test Containers 4.0.1, released 2021-07-29
+
+Code name: Allow dots in release candidate suffix
+
+## Summary
+
+Release 4.0.1 contains a small update that allows using dots as separators for release candidate versions.
+
+We also disabled the `withReuse()` feature for Exasol versions below 7, where it did not work reliably.
+
+## Features
+
+* #152: Version parser now accepts suffixes starting with a dot
+
+## Dependency Updates
+
+### Compile Dependency Updates
+
+* Updated `com.exasol:bucketfs-java:2.0.1` to `2.1.0`
```

Comment:
did you update all dependencies?

Reply:
Except for the JDBC driver for which now a 7.1 version is also available. But I want to do 7.1 compatibility separate. A small smoke test showed that there is probably a loot to do for 7.1